### ISSUE TITLE
Model Accuracy Is Low

### ISSUE THREAD

**Issue by KudahShambare** [KudahShambare/Machine-Learning-With-Python#1]
I need to play around with parameters to archive a better accuracy score

**Comment by KudahShambare** [comment]
Adjusted my neural network

# Kerras Sequential model

model = keras.Sequential()
model.add(Dense(128, input_dim=13, activation='relu', name="H1" ))
model.add(Dense(64, activation='relu', name="H2"))
model.add(Dense(32, activation='relu', name="H3"))
model.add(Dense(16, activation='relu', name="H4" ))
model.add(Dense(3, activation='softmax',name="Output"))
